### ISSUE TITLE
Download category icon bug

### ISSUE THREAD

**Issue by KiraTheUltimate** [e107inc/e107#3501]
Hi!
If I want to upload a download category icon image, I can't find the uploaded image in the list. But in Media Manager there is the image, but "None selected" category.

version: git.

![01](https://user-images.githubusercontent.com/12445784/46906606-68cfc380-cf06-11e8-9fc6-e0d38810b791.JPG)

**Comment by KiraTheUltimate** [comment]
@rica-carv 
I think not. That issue is OK now. The problem now is, that the image don't appear on the image picker platform after I uploaded, so i can't set up the download category.

**Comment by CaMer0n** [comment]
@KiraTheUltimate If you go to /e107_admin/image.php (media-manager) do you see the icon image you uploaded?

**Comment by KiraTheUltimate** [comment]
@CaMer0n 
Yes, I see it, just not show on the picker page of the download category.

**Comment by CaMer0n** [comment]
@KiraTheUltimate Please test with the changes. Thank you.

**Comment by CaMer0n** [comment]
@KiraTheUltimate Great! Thank you!
